<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPainter</class>
 <widget class="QDialog" name="DialogPainter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="CPainter" name="wContent" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="wOption" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Color</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnBlack">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(0, 0, 0);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(0, 0, 0);};</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGray">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(80,80, 80);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(80, 80, 80);};</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRed">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(255,0, 0);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(255, 0, 0);};</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGreen">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(0,255, 0);}
QPushButton:checked{border-radius: 15px;  border:8px groove gray; border-style: outset;background-color: rgb(0, 255, 0);};</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBlue">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(0,0, 255);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(0, 0, 255);};</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnWhite">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(255,255, 255);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(255, 255, 255);};</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Pen</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnS">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(225,225, 225);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(180, 180, 180);};</string>
           </property>
           <property name="text">
            <string>A</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnM">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(225,225, 225);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(180, 180, 180);};</string>
           </property>
           <property name="text">
            <string>A</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnL">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ border-radius: 15px;  border: 1px groove gray; border-style: outset;background-color: rgb(225,225, 225);}
QPushButton:checked{border-radius: 15px;  border: 8px groove gray; border-style: outset;background-color: rgb(180, 180, 180);};</string>
           </property>
           <property name="text">
            <string>A</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>40</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>送出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPainter</class>
   <extends>QWidget</extends>
   <header>DialogPainter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
